.github/workflows/CI_build.yml

```diff
@@ -0,0 +1,235 @@
+name: CI_build
+
+on: [push, pull_request]
+
+jobs:
+
+  # failing with D:\a\AusweisApp\AusweisApp\test\helper\common\MockReleaseInformation.cpp(42,35): error C2248: 'QString::QString': cannot access private member declared in class 'QString' [C:\_build\test\helper\common\AusweisAppTestHelperCommon.vcxproj]
```

Comment:
Please remove the comment. Seems you tried to build unit tests in release mode. That is not supported at the moment.

Reply:
Changed to debug build to overcome this problem as the QT Test component is part of the QT installation and automatically triggers the test build.